### PR TITLE
Routes to Github CB-1

### PR DESCRIPTION
**Pull request by keziaabraham**
* **Please check if the PR fulfills these requirements**
- [ ] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
 This is my first pull request however I had to modify the delete route.

* **What is the current behavior?What does the code do before you made the new changes?** 
It contains 3 routes: post, get, and delete, which allows a person to make changes to the API database.

* **What is the new behavior?**
The delete route is now able to delete a person based on the ID number that is passed as a query parameter.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
This PR does not introduce a breaking change. There are no new commands that need to be run.

* **Other information**:

### CODE REVIEW

File: KeziaA/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        pResponse.status(500).json({'message': 'There was an error' })
+    })
+
+
+})
+//route used to get a list of people in the database
+app.get('/getAllPeople',function(pRequest,pResponse)
+{
+    const body = pRequest.body
+    axios.get('http://java-sample-api-2020.herokuapp.com/getAllPeople',body)//gets all the people in the database
```
Comment (NathanP9000) [inline KeziaA/index.js]: When getting people the body parameter is not needed.

**Review by NathanP9000 — CHANGES_REQUESTED** [review]
Everything looks functional, and the code looks nice, but on line 27 the body parameter should probably be removed as the get request does not need the body.

**Review by iakujobi — COMMENTED** [review]
- Modify the regular functions to arrow functions in lines 7, 11, 24, 28, 40, and 45. 

- Remove the body parameters in the get and delete routes

**Comment by sanjanajadhavv** [comment]
The code looks really organized and easy to understand.
- body parameters are only needed in the post route

File: KeziaA/index.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+const express = require('express')
```
Comment (sanjanajadhavv) [inline KeziaA/index.js:1]: The code looks really organized and easy to understand.
- body parameters are only needed in the post route

**Review by sanjanajadhavv — CHANGES_REQUESTED** [review]

File: KeziaA/index.js (hunk 2 above)
Comment (sanjanajadhavv) [inline KeziaA/index.js:1]: The code looks really organized and easy to understand.
- body parameters are only needed in the post route

**Review by NathanP9000 — DISMISSED** [review]
The code looks good now.

**Review by iakujobi — CHANGES_REQUESTED** [review]
The code is good, but there is just one minor error on lines 43 and 44. They are not aligned properly.

**Review by iakujobi — APPROVED** [review]
Code is ready to be merged. Nice job!

**Review by sanjanajadhavv — APPROVED** [review]
Looks good!

**Pull request merged**